g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb

```diff
@@ -442,7 +442,7 @@
       "source": [
         "results = base_model.evaluate(test_dataset)\n",
         "named_results = dict(zip(base_model.metrics_names, results))\n",
-        "print('\\naccuracy:', named_results['accuracy'])"
+        "print('\\naccuracy:', named_results['acc'])"
```

Comment:
Thanks for the PR, @mo-zahran. 

The metric names (taken from `base_model.metrics_names`)  are usually the same as the `metrics` arg in the `base_model.compile()` call, plus some loss terms. Since we specify `base_model.compile(..., metrics=['accuracy'])` in the previous code block, the reported metric name should be `'accuracy'`. When running this notebook on Google Colab [here](https://colab.research.google.com/github/tensorflow/neural-structured-learning/blob/master/g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb), the metric names are `['loss', 'accuracy']`.

Would you mind sharing your environment setting (e.g. where you run this notebook and which TF version you use), and your local changes to the notebook (if any)?

Thanks.

Reply:
After deeper investigation, I learned that the metric name actually depends on the TensorFlow version. On TF 2 (which is the default in this notebook), the metric would be named as `'accuracy'`. However on TF 1, the metric would be named as `'acc'` even though the model is compiled with `metrics=['accuracy']`.

One way to make the notebook work for both TF versions is to compile the model with `metrics=['acc']`, which names the metric as `'acc'` on both TF 1 and 2. I am going to accept this PR and make subsequent changes to the compilation arguments.

Thanks again for the PR and bringing this issue up!